<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1343</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ALPD</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>30</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
box-shadow: 55px 10px #888888;

</string>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Images Folder</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Output Folder</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="inputPath">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="outputPath">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="skipButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>650</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#6849D0</string>
    </property>
    <property name="text">
     <string>Skip</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>650</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#6849D0</string>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QLabel" name="mainImage">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>100</y>
      <width>421</width>
      <height>291</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="plateImg">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>420</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="seg1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>459</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="seg2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>459</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="seg3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>469</y>
      <width>51</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QLabel" name="seg4">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>470</y>
      <width>51</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QLabel" name="seg5">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>450</y>
      <width>71</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QLabel" name="seg6">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>440</y>
      <width>20</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QLabel" name="seg7">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>440</y>
      <width>61</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="val1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>540</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="val2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>540</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="val3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>540</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="val4">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>540</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="val5">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>540</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="val6">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>540</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="val7">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>540</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>660</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:15px</string>
    </property>
    <property name="text">
     <string>Enter 0 for unwanted characters</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1343</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
